### PR TITLE
fix: SMART_WALLET account flag for smart wallet vs. ag-solo client

### LINKED ISSUE #5783: Invitations are missing on instagoric using the default propsals
## Describe the bug
Invitations are missing on instagoric using the default propsals.  This was working but seems to be broken with the latest 44c34cbdf936e86a8872622acb0e5c733f927f75

## To Reproduce
1. Stand up instagoric
2. no zoe invites in the ag-solo-manual-0 are received

## Expected behavior
zoe invites should be populated

https://github.com/Agoric/instagoric/blob/main/bases/shared/resources/economy-proposals.json

### PR DESCRIPTION
**Pull request by dckc**
quick hack based on discussion with @warner ; completely untested.
cc @turadg @arirubinstein 

REMOTE_WALLET is really a new feat, but w.r.t. the smart wallet
depositFacet conflict in #5783, this is (the start of) a fix.

  - [x] TODO: default to ag-solo compatible remote client wallet
          during a transition to smart-wallet-default

closes: #5783
refs: #4654

### CODE REVIEW

**Comment by dckc** [comment]
@arirubinstein is it straightforward to test this `5783-remote-wallet-flag` branch in a similar configuration to where we saw the missing invitations? If so, would you please?

**Comment by arirubinstein** [comment]
Yes, I can rev loadtest with an image built from `5783-remote-wallet-flag`

**Comment by dckc** [comment]
Thanks for spinning that up; fix confirmed:

```
command[4]
E(history[3]).getCurrentAmount()
history[4]
{"brand":[Object Alleged: Zoe Invitation brand]{},"value":[{"description":"oracle invitation","handle":[Object Alleged: InvitationHandle]{},"installation":[Object Alleged: BundleInstallation]{},"instance":[Object Alleged: InstanceHandle]{}},{"description":"econCommitteeCharter noop","handle":[Object Alleged: InvitationHandle]{},"installation":[Object Alleged: BundleInstallation]{},"instance":[Object Alleged: InstanceHandle]{}},{"description":"Voter0","handle":[Object Alleged: InvitationHandle]{},"installation":[Object Alleged: BundleInstallation]{},"instance":[Object Alleged: InstanceHandle]{}}]}
```

File: packages/vats/src/core/basic-behaviors.js [hunk 1]
```diff
@@ -207,6 +207,11 @@ export const makeAddressNameHubs = async ({
 };
 harden(makeAddressNameHubs);
 
+const AccountFlags = /** @type {const} */ ({
+  SMART_WALLET: 'SMART_WALLET', // TODO: make this the default
+  REMOTE_WALLET: 'REMOTE_WALLET',
```
Comment (turadg) [inline packages/vats/src/core/basic-behaviors.js]: consider commenting that this is for ag-solo client

File: packages/vats/src/core/basic-behaviors.js [hunk 2]
```diff
@@ -207,6 +207,11 @@ export const makeAddressNameHubs = async ({
 };
 harden(makeAddressNameHubs);
 
+const AccountFlags = /** @type {const} */ ({
```
Comment (turadg) [inline packages/vats/src/core/basic-behaviors.js]: from discussion, these are `PowerFlags`. best to keep the name consistent. though it's worth a comment here that this enum is a subset of all known power flags.

**Review by turadg — APPROVED** [review]

**Pull request merged**